Formula/geant4.rb

```diff
@@ -66,13 +69,13 @@ class Geant4 < Formula
   end
 
   resource "G4ENSDFSTATE" do
-    url "https://cern.ch/geant4-data/datasets/G4ENSDFSTATE.2.2.tar.gz"
-    sha256 "dd7e27ef62070734a4a709601f5b3bada6641b111eb7069344e4f99a01d6e0a6"
+    url "https://cern.ch/geant4-data/datasets/G4ENSDFSTATE.2.3.tar.gz"
+    sha256 "9444c5e0820791abd3ccaace105b0e47790fadce286e11149834e79c4a8e9203"
   end
 
   resource "RealSurface" do
-    url "https://cern.ch/geant4-data/datasets/G4RealSurface.2.1.1.tar.gz"
-    sha256 "90481ff97a7c3fa792b7a2a21c9ed80a40e6be386e581a39950c844b2dd06f50"
+    url "https://cern.ch/geant4-data/datasets/G4RealSurface.2.2.tar.gz"
+    sha256 "9954dee0012f5331267f783690e912e72db5bf52ea9babecd12ea22282176820"
   end
 
   def install
```

Comment:
Can you try adding:

```ruby
# Work around "error: no member named 'signbit' in the global namespace"
    ENV.delete("SDKROOT") if DevelopmentTools.clang_build_version >= 900
```

Just to see if the same thing as ROOT works?

Reply:
Thanks, I've tried that with a fresh reinstall and it makes no difference to either building or installing. The formula builds/installs fine with or without this. The test still fails in both cases, and explicitly removing `SDKROOT` in the test env doesn't affect things.

There's something in the env that's set up for `test` that's confusing CMake, so I'll try and narrow this down.

Reply:
Does the cmake file you use at the start of the test block come with the installation tarball, or is it something that's made during the install process?

If it's the latter, maybe it's inheriting stuff from Homebrew's build environment that makes it unsuitable for use outside it.

Reply:
The cmake file is from one of the example programs, so direct from the tarball without modification.

Reply:
Hmmm. Odd. Ok, it may be worth trying to run the test in the install block to see if it behaves differently; maybe it's not necessarily the environment that's the culprit. (Or maybe it is and you know you just need to replicate the build environment.)